### PR TITLE
Sprints/30secondstomars

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Solution/Solution/Entities/Contact/Entity.xml [hunk 1]
```diff
@@ -1158,7 +1158,7 @@
               </option>
               <option value="3">
                 <labels>
-                  <label description="Bilingual" languagecode="1033" />
+                  <label description="No Preference" languagecode="1033" />
                   <label description="Bilingue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/Entity.xml:1162]: missing french translation

File: src/Solution/Solution/Entities/Contact/Entity.xml [hunk 2]
```diff
@@ -1179,11 +1179,11 @@
             </options>
           </optionset>
           <displaynames>
-            <displayname description="Language" languagecode="1033" />
+            <displayname description="Preferred Language" languagecode="1033" />
             <displayname description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/Entity.xml:1183]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{6b5ff2ea-9232-ec11-b6e6-000d3a09fd9f}.xml [hunk 3]
```diff
@@ -89,12 +87,12 @@
                       </cell>
                     </row>
                     <row>
-                      <cell id="{921eaba2-6044-405e-8354-c3c5dbe3e97f}" labelid="{2443bd31-1e3b-4a44-93b5-590af4126c0c}">
+                      <cell id="{583eaecf-dd2e-43c1-becb-432792dc2ee5}" locklevel="0" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Job Title" languagecode="1033" />
-                          <label description="Fonction" languagecode="1036" />
+                          <label description="Preferred Language" languagecode="1033" />
+                          <label description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{6b5ff2ea-9232-ec11-b6e6-000d3a09fd9f}.xml:93]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{7c2799bf-9232-ec11-b6e6-000d3a09f410}.xml [hunk 4]
```diff
@@ -91,12 +89,12 @@
                       </cell>
                     </row>
                     <row>
-                      <cell id="{e456e162-28a1-1432-05a6-b03b4528e847}" showlabel="true" locklevel="0" labelid="{cfdf0d67-67d1-4892-9df3-56e6a63d1983}">
+                      <cell id="{e4eab838-9b08-41f1-a32a-e3a571094440}" locklevel="0" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Job Title" languagecode="1033" />
-                          <label description="Fonction" languagecode="1036" />
+                          <label description="Preferred Language" languagecode="1033" />
+                          <label description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{7c2799bf-9232-ec11-b6e6-000d3a09f410}.xml:95]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml [hunk 5]
```diff
@@ -118,12 +118,12 @@
                       </cell>
                     </row>
                     <row>
-                      <cell id="{7284da39-112d-44f2-be81-93c1ae483258}" showlabel="true" locklevel="0">
+                      <cell id="{a0771b23-9eb1-4560-b962-7d1125a2638e}" locklevel="0" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Preferred Language of Communication" languagecode="1033" />
-                          <label description="Langue de communication privilégiée" languagecode="1036" />
+                          <label description="Preferred Language" languagecode="1033" />
+                          <label description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml:124]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/quick/{b7048da0-fa4b-ed11-bba1-0022486df7f0}.xml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                <section id="AD49B5BF-256F-46A8-9C75-947146172EB9" name="section_preferredcommunication" columns="1" showlabel="false" showbar="false" IsUserDefined="0" labelwidth="130" labelid="{65222d15-16ff-4eb1-b3ef-86cc11eb4bb3}">
+                  <labels>
+                    <label description="Preferred Communication" languagecode="1033" />
+                  </labels>
+                  <rows>
+                    <row>
+                      <cell id="{fca40826-27de-4292-9f76-3ef5c83417aa}" locklevel="0" colspan="1" rowspan="1">
+                        <labels>
+                          <label description="Preferred Language" languagecode="1033" />
+                          <label description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/quick/{b7048da0-fa4b-ed11-bba1-0022486df7f0}.xml:26]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/quickCreate/{7d790f4b-9332-ec11-b6e6-000d3a09f410}.xml [hunk 7]
```diff
@@ -47,12 +47,12 @@
                       </cell>
                     </row>
                     <row>
-                      <cell id="{44bca3e0-9296-db96-4290-9bd61e9670c9}" showlabel="true" locklevel="0" labelid="{22de325f-d1d9-4c77-93cf-5f409a3bd0a4}">
+                      <cell id="{2eb68536-3088-4cd4-b0c4-c1b94a76367d}" locklevel="0" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Language" languagecode="1033" />
+                          <label description="Preferred Language" languagecode="1033" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/quickCreate/{7d790f4b-9332-ec11-b6e6-000d3a09f410}.xml:52]: missing french translation

File: src/Solution/Solution/Entities/Contact/FormXml/quickCreate/{ec29b747-fbc2-44a1-97ef-b2e70e4b72cf}.xml [hunk 8]
```diff
@@ -69,6 +69,15 @@
                     <label description="Informations sur le contact" languagecode="1036" />
                   </labels>
                   <rows>
+                    <row>
+                      <cell id="{5b710535-b725-45be-9063-8ad05c90b94d}" locklevel="0" colspan="1" rowspan="1">
+                        <labels>
+                          <label description="Preferred Language" languagecode="1033" />
+                          <label description="Langue" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/quickCreate/{ec29b747-fbc2-44a1-97ef-b2e70e4b72cf}.xml:76]: missing french translation

File: src/Solution/Solution/Entities/opc_complaint/FormXml/main/{b8bed16c-fe15-4801-ae58-2636ceecce77}.xml [hunk 9]
```diff
@@ -254,12 +254,17 @@
                       </cell>
                     </row>
                     <row>
-                      <cell id="{b7069f32-7d08-4785-91c5-f90a7ee7f708}" showlabel="true" locklevel="0">
+                      <cell locklevel="0" id="{fcfe0189-e1da-4972-89b5-deeca4483c3d}" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Complainant Legal Representative Firm" languagecode="1033" />
-                          <label description="Cabinet du représentant légal du plaignant" languagecode="1036" />
+                          <label description="Preferred Communication" languagecode="1033" />
+                          <label description="Plaignant" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_complaint/FormXml/main/{b8bed16c-fe15-4801-ae58-2636ceecce77}.xml:260]: missing french translation

File: src/Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml [hunk 10]
```diff
@@ -346,19 +346,16 @@
                       </cell>
                     </row>
                     <row>
-                      <cell auto="false" showlabel="true" locklevel="0" rowspan="1" colspan="1" id="{4828e4ae-6040-4c94-9459-dae4bf25b2f4}">
+                      <cell locklevel="0" id="{7e68cf96-7e07-4885-aace-fd15b6a82b1d}" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Representative" languagecode="1033" />
-                          <label description="Représentant" languagecode="1036" />
+                          <label description="Preferred Communication" languagecode="1033" />
+                          <label description="Plaignant" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml:352]: <IntroducedVersion>1.0.357</IntroducedVersion>

File: src/Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml [hunk 11]
```diff
@@ -346,19 +346,16 @@
...
                         </labels>
-                        <control id="opc_representativeid" classid="{270BD3DB-D9AF-4782-9025-509E298DEC0A}" datafieldname="opc_representativeid" disabled="false">
+                        <control id="qv_preferredcommunication" classid="{5C5600E0-1D6E-4205-A272-BE80DA87FD42}" datafieldname="opc_complainantid">
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pafile/FormXml/main/{e1d2d24d-402c-4df8-8476-f592ffa5753b}.xml:354]: @mitchkirady wrong prefix?

File: src/Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml [hunk 12]
```diff
@@ -345,19 +345,15 @@
                       </cell>
                     </row>
                     <row>
-                      <cell auto="false" showlabel="true" locklevel="0" rowspan="1" colspan="1" id="{e3a0b75b-2db1-43c2-910b-3717525caac1}">
+                      <cell locklevel="0" id="{8208c24e-eec2-4bfb-8b21-3887df11cb2f}" colspan="1" rowspan="1">
                         <labels>
-                          <label description="Representative" languagecode="1033" />
-                          <label description="Représentant" languagecode="1036" />
+                          <label description="Preferred Communication" languagecode="1033" />
+                          <label description="Plaignant" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml:351]: missing french translation

File: src/Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml [hunk 13]
```diff
@@ -413,10 +409,21 @@
                       </cell>
                     </row>
                     <row>
-                      <cell locklevel="0" id="{6a4fd765-e311-4acb-b52c-a001eecd3986}">
+                      <cell auto="false" showlabel="true" locklevel="0" rowspan="1" colspan="1" id="{f9e42568-d3a3-40d3-82ac-536e5a48380e}">
                         <labels>
-                          <label description="" languagecode="1033" />
+                          <label description="Respondent" languagecode="1033" />
+                          <label description="Intimé" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pipedafile/FormXml/main/{e44223e0-9cac-4344-b608-82e51cc2e4d0}.xml:415]: Should be "Intervenant" I believe here... to be confirmed.

File: src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{1308f67b-9549-ed11-bba1-0022486df7f0}.xml [hunk 14]
```diff
@@ -0,0 +1,64 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{1308f67b-9549-ed11-bba1-0022486df7f0}.xml:1]: Where is this coming from?

File: src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{2b65175a-8f49-ed11-bba1-0022486df7f0}.xml [hunk 15]
```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{2b65175a-8f49-ed11-bba1-0022486df7f0}.xml:1]: Where is this coming from?

File: src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{3554b978-9749-ed11-bba2-0022486df95f}.xml [hunk 16]
```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{3554b978-9749-ed11-bba2-0022486df95f}.xml:1]: where is this coming from?

Comment (mitchkirady) [reply src/Solution/Solution/Entities/opc_pipedafile/SavedQueries/{3554b978-9749-ed11-bba2-0022486df95f}.xml:1]: dashboard queries

File: src/WebResources/typings/xrmdefinitelytyped/Form/contact/Quick/PreferredCommunication.d.ts [hunk 17]
```diff
@@ -0,0 +1,96 @@
+declare namespace Form.contact.Quick {
+  namespace PreferredCommunication {
```
Comment (pierluc-ssoucy) [inline src/WebResources/typings/xrmdefinitelytyped/Form/contact/Quick/PreferredCommunication.d.ts:2]: preferred way of communication feels like an odd name for a quick create form.

**Pull request merged**